db/data_migrations/20221202133634_assign_existing_news_to_trade_news.rb

```diff
@@ -0,0 +1,19 @@
+Sequel.migration do
+  # IMPORTANT! Data migrations should be Idempotent, they may get re-run as part
+  # of data rollbacks
+
+  up do
+    trade_news = News::Collection.find_or_create(name: 'Trade news') do |collection|
```

Comment:
I'm wondering whether this should be a rake task if we don't want it to be rolled back ever?

Reply:
And maybe add an explicit callback to force all items to have the trade_news collection?

Reply:
There's a separate PR for that - may roll out tomorrow but more likely on thursday once I know the data is straight

As regards rake task - would normally agree _but_ we'll end up with all news stories disappearing from the site including the home page block because they come to depend upon belonging to a published collection